ansible/roles/ooni-bouncer/templates/bouncer.metayaml

```diff
@@ -0,0 +1,2 @@
+{# meta-template to ensure that the templated configuration file is at least valid yaml #}
+{{ lookup('template', 'bouncer.yaml.j2') | from_yaml | to_nice_yaml }}
```

Comment:
I now understand why above you call it `metayaml` instead of `.j2`

Reply:
I just did not care about extension for the file that has just two lines, so I just added something that is one bit better than `bouncer.blah`. :man_shrugging: 

The only reason for me to specify extensions in those templates is to get syntax highlighting for the "base-format".